index.js

```diff
@@ -0,0 +1,67 @@
+import Book from './modules/Book.js';
+import Library from './modules/Library.js';
+import list from './modules/components/List.js';
+import add from './modules/components/Add.js';
+import contact from './modules/components/Contact.js';
+import { DateTime } from './modules/luxon.js';
+
+const app = document.querySelector('#app');
+const dateTime = document.querySelector('#date-time');
+
+const library = new Library();
+
+function loadBooks() {
+  app.innerHTML = '';
+  const listElement = document.createRange().createContextualFragment(list(library));
+  app.append(listElement);
+  const booksList = document.querySelector('#book-list');
+  booksList.innerHTML = '';
+  library.load(booksList);
+}
```

Comment:
- [ ] Kindly refactor your methods and functions to arrow functions. as required from the requirements.